package.json

```diff
@@ -3,13 +3,31 @@
   "version": "0.15.5",
   "main": "build/src/index.js",
   "types": "build/src/index.d.ts",
+  "description": "A Web3 library for interacting with the ZkSync Layer 2 scaling solution.",
   "files": [
     "build/",
     "abi/",
     "src/",
     "typechain/"
   ],
   "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matter-labs/zksync-web3.git"
+  },
+  "bugs": {
+    "url": "https://github.com/matter-labs/zksync-web3/issues"
```

Comment:
The same goes for issues: the common issue page is used for both SDK implementations. The issue template should provide an option to select which SDK implementation is related to the issue.